usr/sbin/p11sak/p11sak.c

```diff
@@ -2625,6 +2893,9 @@ int main(int argc, char *argv[])
         goto done;
     }
 
+    /* Parse p11sak_defined_attrs.conf */
+    rc = parse_file();
```

Comment:
Store to `rc` is unused here.  Should you care about the return code or is the error already reported to the user and the program can continue despite possible errors?

Reply:
I would say it should stop if parsing the config file fails. parse_file did already print the error messages, so a simple `goto done` or `return ` might do it.

Reply:
Done